PublishedModules/glueckself/mpdclient/mpdhelper.py

```diff
@@ -0,0 +1,225 @@
+# Copyright 2006-2009 Scott Horowitz <stonecrest@gmail.com>
```

Comment:
If you are distributing the API, you don't need to specify it in the module install file. Having it specified in the install file will make Alice download the api and install it

Reply:
I copied the from Sonata, a GUI MPD client. Am I right that I need to keep the copyright header?

Reply:
The requirements are for the base mpd library.
It goes python-mpd2 (via requirements) - mpdhelper.py - mpdclient.py

I don't really understand how to use the mpd2 library, even after reading mpdhelper.py. To be honest, I also don't understand how the mpdhelper.MPDClient class provides the play()/stop()/... methods, but I've seen them used in Sonatas code (I guess they are added with setattr?). That's why I'm using the helper.

Reply:
Ooh, sorry, did not understand it that way! So it's ok :)